dashboard/app/controllers/courses_controller.rb

```diff
@@ -20,4 +29,21 @@ def show
 
     render 'show', locals: {course: course}
   end
+
+  def new
+    render 'new'
+  end
+
+  def create
+    course = Course.new(name: params.require(:course).require(:name))
```

Comment:
Worth pulling this out to a private `course_params` method, and re-using in the `show` action?  For example: [`ScriptsController#script_params`](https://github.com/code-dot-org/code-dot-org/blob/843100968270e22522d9245aa57ec69d8f3c5fd1/dashboard/app/controllers/scripts_controller.rb#L98-L100).